pubspec.yaml

```diff
@@ -37,7 +37,7 @@ dev_dependencies:
   grinder: "^0.8.0"
   http: ">=0.11.0 <0.13.0"
   js: "^0.6.0"
-  node_preamble: "^1.1.0"
+  node_preamble: "^1.4.5"
```

Comment:
I would prefer to leave this as-is. We aren't relying on any *API features* of more recent preamble versions, we're just taking advantage of bug fixes.

Reply:
reverting, although I could really use a `sass.dart.js` that has 1.4.5+, as it fixes several issues I've encountered with 1.1.0.

Reply:
We'll download the latest version regardless of whether the version range forces it to be downloaded, unless doing so would cause a version conflict.